lib/impl/AzureBlobFiles.js

```diff
@@ -334,10 +335,14 @@ class AzureBlobFiles extends Files {
    * @override
    * @private
    */
-  _getUrl (filePath) {
+  _getUrl (filePath, urlType = UrlType.public) {
```

Comment:
it might make sense to use require('url') here instead of all this string manipulation